crates/router/src/consts.rs

```diff
@@ -91,3 +91,6 @@ pub const MAX_SESSION_EXPIRY: u32 = 7890000;
 pub const MIN_SESSION_EXPIRY: u32 = 60;
 
 pub const LOCKER_HEALTH_CALL_PATH: &str = "/health";
+
+// URL for checking the outgoing call
+pub const OUTGOING_CALL_URL: &str = "https://api.stripe.com/healthcheck";
```

Comment:
There is still an edge case, where only this connector is down, we need to come up with a more permanent fix for this

Reply:
This can be resolved as a separate issue